resistbot-gateway-forwarder/src/index.ts

```diff
@@ -46,7 +51,7 @@ client.on(Events.InteractionCreate, async (interaction) => {
     }).catch((e) => {
         console.error('Got error for interaction', e)
     });
-    interaction.reply({ content: 'Pong', ephemeral: true })
+    // interaction.reply({ content: 'Pong', ephemeral: true })
```

Comment:
something we can just delete?

Reply:
I left this in for now because without it, the interaction in discord will say it failed. UX will need a pass once things are working